src/database/ghost.js

```diff
@@ -12,6 +12,7 @@ module.exports = knex => {
       table.string('folder')
       table.string('file')
       table.text('content')
+      table.boolean('deleted').defaultTo(false)
```

Comment:
You don't need it here, it's created by the migration now

Reply:
Removed it, good point.